crates/zed/src/languages/svelte/config.toml

```diff
@@ -15,6 +15,10 @@ brackets = [
 scope_opt_in_language_servers = ["tailwindcss-language-server"]
 prettier_parser_name = "svelte"
 
+[overrides.comment]
+line_comment = { remove = true }
+line_comments = "// "
```

Comment:
I don't think you need this `overrides.comment` section. Could you remove it, and confirm that commenting / uncommenting still works as expected in Svelte?

Reply:
You would be correct! Looks like it was just the `block_comment` :+1: